translations/en.json

```diff
@@ -1125,7 +1126,10 @@
 	"custom-seed-phrase-text-2": "some words are misspelled.",
 	"custom-seed-phrase-text-3": " If so, you'll end up creating a",
 	"custom-seed-phrase-text-4": " new account",
-	"to-enable-biometric": "To enable {{bio-type-label}}, your must save your password on the unlock screen",
+	"to-enable-biometric-1": "To enable {{bio-type-label}}, your must ",
```

Comment:
I've discussed this with @andmironov , it will be a nightmare for translators, and translations will be awful, so better to do not split sentences